plugin.video.vrt.nu/channels.m3u8

```diff
@@ -0,0 +1,87 @@
+#EXTM3U
```

Comment:
Are all of these stream freely available to the public? Where are the sources?

Reply:
These are freely available sources: https://www.vrt.be/nl/aanbod/kijk-en-luister/radio-luisteren/streamingslinks-radio/